package.json

```diff
@@ -41,10 +42,12 @@
     "@typescript-eslint/parser": "^4.28.2",
     "babel-jest": "^27.0.6",
     "eslint": "^7.30.0",
+    "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-jest": "^24.3.6",
+    "eslint-plugin-prettier": "^3.4.0",
     "jest": "^27.0.6",
     "nyc": "^15.1.0",
-    "prettier": "^2.2.1",
+    "prettier": "2.3.2",
```

Comment:
any reason not to use `^2.3.2`?

Reply:
Making sure that everyone will get the same version, because even small patches in prettier can make a some difference.